libgit/autogit_manager.go

```diff
@@ -307,6 +306,10 @@ func (am *AutogitManager) makeCloningFile(
 // tramples it, so the caller must ensure that they are requesting a
 // clone from the same repo/branch as the existing repo that might
 // have already been there.
+//
+// If the caller specifies a non-master `branchName`, they should make
```

Comment:
Hrm how did this line not starting with "Clone ..." pass the linter? Maybe we aren't linting this package.

Reply:
It's not the first line in the comment.  Only the first paragraph matters,

Reply:
Oh I see; I totally missed the previous lines..